inc/functions.php

```diff
@@ -1801,7 +1801,10 @@ function get_post_icons()
 		eval("\$iconlist .= \"".$templates->get("posticons_icon")."\";");
 	}
 
-	eval("\$posticons = \"".$templates->get("posticons")."\";");
+	if(!empty($iconlist))
```

Comment:
We should add an else statement to set `$posticons` to an empty string or such, otherwise it just shows `Array`.